observability/distr_tracing/distr-tracing-tempo-configuring.adoc

```diff
@@ -1,38 +1,43 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="distr-tracing-tempo-configuring"]
-= Configuring
 include::_attributes/common-attributes.adoc[]
+[id="distr-tracing-tempo-configuring"]
+= Configuring the {DTShortName}
 :context: distr-tracing-tempo-configuring
 
 toc::[]
 
+[TIP]
+====
+For information about configuring the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc#distr-tracing-jaeger-configuring[Configuring] in the {JaegerShortName} documentation.
+====
+
 The {TempoOperator} uses a custom resource definition (CRD) file that defines the architecture and configuration settings for creating and deploying the {TempoShortName} resources. You can install the default configuration or modify the file.
 
 ifdef::openshift-enterprise,openshift-dedicated[]
 [id="configuring-storages_{context}"]
 == Configuring back-end storage
 
-For information about configuring the back-end storage, see xref:../../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage] and the relevant configuration section for your chosen storage option.
+For information about configuring the back-end storage, see xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[Understanding persistent storage] and the relevant configuration section for your chosen storage option.
 endif::[]
 
 include::modules/distr-tracing-tempo-config-default.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing a TempoStack instance]
-* xref:../../../observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc#dist-tracing-tempo-installing[Installing a TempoMonolithic instance]
+* xref:../../observability/distr_tracing/distr-tracing-tempo-installing.adoc#distr-tracing-tempo-installing[Installing a TempoStack instance]
```

Comment:
@max-cx Some of these friend reference texts actually link to different titles. I can tell the content is the same, but I think at some point they might need renamed. For example, when I click "Installing a TempoStack instance," it links me to "Installing the Distributed Tracing Platform". AFAICT this does install a tempostack instance, but that isn't the name of the section. 

Note that there are xrefs above this module that do that, as well.

Reply:
:slightly_smiling_face: These are modules, and AFAIK I can't link to a module, so I link to the assembly and use the module heading as the link text.